client/src/main/java/tds/accommodation/Dependency.java

```diff
@@ -0,0 +1,141 @@
+package tds.accommodation;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents an accommodation/tool dependency
+ */
+public class Dependency {
```

Comment:
I vote AccommodationDependency in that case (since we already seem to be lumping "tools" into that category)